app/views/what_kind_of_proof/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :form_card_title, "What do you have?" %>
+
+<% content_for :form_card_body do %>
+    <%= fields_for(:form, @form, builder: CfaFormBuilder) do |f| %>
```

Comment:
Hey, this looks great! It occurs to me that we have this pattern already in Michigan (checkbox_set_with_none in the MbFormBuilder) and we may want to pull over or make this more generic, though, since we'll be using this pattern again in the future. I think it makes sense to do that now—happy to pair on it tomorrow.

Here are a few links about how we've done it in the past, and how we may want to consider making it a more generic piece of the formbuilder:

MbFormBuilder method: https://github.com/codeforamerica/michigan-benefits/blob/4a987abf226bc30d5688369f40b315510786d92b/app/helpers/mb_form_builder.rb#L343

application.js.erb : https://github.com/codeforamerica/michigan-benefits/blob/1f54952b82e52d845f3a0d19817d00d0ffe0ea3f/app/assets/javascripts/application.js.erb#L211

And here's an example of how we've used the formbuilder method in a view: https://github.com/codeforamerica/michigan-benefits/blob/a1c4ec376a93f3c1105ad9339a23c84cbb9223e2/app/views/integrated/housing_expenses/edit.html.erb

I think this is something we'll want to pull into the styleguide, too, as part of this story: https://www.pivotaltracker.com/story/show/160664826

Reply:
One difference to note from Michigan is that `checkbox_set_with_none` takes in a collection of values (that saves to the same array field) rather than a set of discrete fields.

Reply:
We pulled the checkbox_set_with_none into the form buildee